README.md

```diff
@@ -17,11 +17,11 @@
 
 ## Usage
 
-1. Open one of the flavours below:
-  - [Latte](https://getnoir.app/theme/?name=Catppuccin%20Latte&bg=EFF1F5&text=4c4f69)
-  - [Frappe](https://getnoir.app/theme/?name=Catppuccin%20Frappe&bg=303446&text=C6D0F5)
-  - [Macchiato](https://getnoir.app/theme/?name=Catppuccin%20Macchiato&bg=24273A&text=CAD3F5)
-  - [Mocha](https://getnoir.app/theme/?name=Catppuccin%20Mocha&bg=1E1E2E&text=CDD6F4)
+1. Open one of the flavors below:
+  - [🌻 Latte](https://getnoir.app/theme/?name=Catppuccin%20Latte&bg=eff1f5&text=4c4f69)
+  - [🪴 Frappé](https://getnoir.app/theme/?name=Catppuccin%20Frappé&bg=303446&text=c6d0f5)
```

Comment:
Opening this link only gives me "Frapp", you'll need to use the non-accented version.